Source/Core/Core/PowerPC/BreakPoints.cpp

```diff
@@ -196,6 +196,15 @@ void MemChecks::Remove(u32 address)
   });
 }
 
+void MemChecks::Clear()
+{
+  Core::RunAsCPUThread([&] {
+    m_mem_checks.clear();
+    JitInterface::ClearCache();
+    PowerPC::DBATUpdated();
```

Comment:
Could you explain why this is required?

Reply:
The bug doesn't occur when memory point is deleted by pressing Delete button (not Clear button), so I compared MemChecks::Remove and MemChecks::Clear.
I don't know how it works internally but compared to Memchecks::Remove, MemChecks::Clear lacks JitInterface::ClearCache and PowerPC::DBATUpdated.

Reply:
Ok, that makes sense.